projects/pytorch/Dockerfile

```diff
@@ -16,46 +16,47 @@
 
 FROM sydr/ubuntu20.04-sydr-fuzz
 
-MAINTAINER Theodor Arsenij Larionov-Trichkine
+# Install build dependencies.
+RUN pip3 install astunparse numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions
 
 # Clone target from GitHub.
-RUN pip3 install astunparse numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions && \
-    git clone https://github.com/pytorch/pytorch/ /pytorch_fuzz && cd /pytorch_fuzz && \
+RUN git clone https://github.com/pytorch/pytorch/ /pytorch
+RUN cd /pytorch && \
     git checkout abc54f93145830b502400faa92bec86e05422fbd && \
     git submodule update --init --recursive --jobs 0
 
 # Copy build script and targets.
-COPY build_*.sh \
-    *.patch \
-    *_fuzz.cc \
-    main-impl.cc \
-    *.toml \
-    patch_exceptions.sh \
-    /
-
-# Apply PyTorch fixes + copy for different targets.
-RUN cd /pytorch_fuzz && \
-    git apply /schema_type_parser-stoll.patch && \
-    cp -r /pytorch_fuzz/ /pytorch_sydr/ && \
-    cp -r /pytorch_fuzz/ /pytorch_cov/
+COPY *.sh \
+     *.patch \
+     *.cc \
+     /
+
+# Apply PyTorch fixes. 
+RUN cd /pytorch && \
+    git apply /schema_type_parser-stoll.patch
 
 # Patch exceptions throwing to avoid unwanted crashes.
-RUN bash patch_exceptions.sh
-WORKDIR /
+RUN ./patch_exceptions.sh
+
+# Fix PyTorch build.
+RUN rm /usr/bin/x86_64-linux-gnu-gcc && \
+    ln -s /usr/bin/clang /usr/bin/x86_64-linux-gnu-gcc
 
 # Build fuzz targets.
-RUN bash build_fuzz.sh
-RUN bash build_sydr.sh
-RUN bash build_cov.sh
+RUN TARGET="afl" bash build.sh
```

Comment:
do we need to call bash here and later, if there is shebang in the script?

Reply:
Done